packages/account/src/Components/file-uploader-container/file-uploader-container.jsx

```diff
@@ -1,10 +1,42 @@
 import React from 'react';
-import { Icon } from '@deriv/components';
+import PropTypes from 'prop-types';
+import { Icon, Text } from '@deriv/components';
 import { Localize } from '@deriv/translations';
 import { WS } from 'Services/ws-methods';
 import FileUploader from './file-uploader.jsx';
 
-const FileUploaderContainer = ({ is_description_disabled, getSocket, onFileDrop, onRef }) => {
+const FileProperties = () => {
+    const properties = [
+        { name: 'size', icon: 'IcPoaFileEightMb', text: 'Less than 8MB' },
+        { name: 'format', icon: 'IcPoaFileFormat', text: 'JPEG  JPG  PNG  PDF  GIF' },
+        { name: 'time', icon: 'IcPoaFileTime', text: '1 - 6 months old' },
+        { name: 'clear', icon: 'IcPoaFileClear', text: 'A clear colour photo or scanned image' },
+        {
+            name: 'with-address',
+            icon: 'IcPoaFileWithAddress',
+            text: 'Issued under your name with your current address',
+        },
```

Comment:
Should the `text` values be `localize`d?

Reply:
> Now that we have added `is_dashboard` to `PlatformContext`, rather than passing it around through props, could each component import the `PlatformContext` instead? This'll save a lot of file changes in this PR.

@aaron-binary Thanks for review this PR.
when I'd start update code I'd imported is_dashboard to each component but there are two reasons that I thought it's better if we pass it as props
1- When the account component rendered we should check if it's the dashboard design or not. checking this condition at the top-level help us to avoid a lot of "IF" statement inside each component 
2- we don't have the account responsive design for the dashboard at the top level, I've checked if it's responsive or not. if we use useContext for each component we should have this condition in every component.

Reply:
- What's the difference between passing it as props vs using the `useContext` hook? 🤔 They both have the same value.
- I'm not sure if I understand your comment, but I would imagine the responsive version for Dashboard's account components would have the same content as the desktop version.